src/main/webapp/quizFunctions.js

```diff
@@ -0,0 +1,31 @@
+// This js file is to implement the quiz functions
+// Music: https://www.bensound.com
+
+let quizQuestions = ["Which plant looks the healthiest?","Which plant is the largest?","Which plant has the most food growing from it?","Which plant needs the most sunlight?","Which plant needs the most water?", "Which plant looks the most serene?", "Which plant is most likely to be poisonous?", "Which plant is likely to grow the fastest?", 
```

Comment:
Can this be declared as a `const` variable? (I don't know javascript very well.)

Reply:
const and var and let are the same thing

Reply:
Not quite. Variables declared with `const` can't be overwritten. Declaring it with `const` signals to the reader that this value is a constant. (It's more of a convention/communication thing than a code-correctness thing.)

See also https://codeburst.io/difference-between-var-let-and-const-in-javascript-fbce2fba7b4